script/deploy-canary-demo

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash
 set -euxo pipefail
+versionInfix=${1+$1}
```

Comment:
Could we rename this `versionInfix` to `version`, I think it is not `Infix` in this context.

Reply:
Just want to clarify a little bit. 
If I'm right,`${1+$1}` returns `$1` only when `$1` exists.
So the goal you want to achieve is:
* By default the version should be empty for latest Canary
* If there's a version number, append it at the end of the demo name

Reply:
Sure, changed

Reply:
> Just want to clarify a little bit. If I'm right,`${1+$1}` returns `$1` only when `$1` exists. So the goal you want to achieve is:
> 
> * By default the version should be empty for latest Canary
> * If there's a version number, append it at the end of the demo name

That's correct